Gemfile

```diff
@@ -30,15 +30,15 @@ gem 'listen', '>= 3.0.5', '< 3.2', group: [:development]
 gem 'mods'
 gem 'negative_captcha'
 gem 'oai', '>=1.1.0'
-gem 'pg', '~> 0.18'
+gem 'pg', '~> 1.1'
```

Comment:
Error:
[ E 2023-03-15 07:18:21.0708 215/T1w age/Cor/App/Implementation.cpp:221 ]: Could not spawn process for application /home/app/webapp: The application encountered the following error: Error loading the 'postgresql' Active Record adapter. Missing a gem it depends on? can't activate pg (~> 1.1), already activated pg-0.21.0. Make sure all dependencies are added to Gemfile. (LoadError)
oral-history-web-1  |   Error ID: e8691954
oral-history-web-1  |   Error details saved to: /tmp/passenger-error-44jDot.html

Needed to upgrade to pg (~> 1.1)